examples/llama-bench/llama-bench.cpp

```diff
@@ -201,6 +218,7 @@ static void print_usage(int /* argc */, char ** argv) {
     printf("  -sm, --split-mode <none|layer|row>  (default: %s)\n", join(transform_to_str(cmd_params_defaults.split_mode, split_mode_str), ",").c_str());
     printf("  -mg, --main-gpu <i>                 (default: %s)\n", join(cmd_params_defaults.main_gpu, ",").c_str());
     printf("  -nkvo, --no-kv-offload <0|1>        (default: %s)\n", join(cmd_params_defaults.no_kv_offload, ",").c_str());
+    printf("  -nmmap, --no-mmap                   (default: %s)\n", cmd_params_defaults.use_mmap ? "0" : "1");
```

Comment:
Print the possible values (`<0|1>`). The parameter is `no-mmap` but the value is `mmap`, the discrepancy needs to be fixed.